### PR TITLE
Move lookup docs that are in druid-proper back into lookups.md

### PR DESCRIPTION
**Pull request by drcrallen**

### CODE REVIEW

**Comment by fjy** [comment]
:-1: We decided during the dev sync that all modules should have their documentation in separate files and have main docs link to those modules. We can discuss that decison more if you don't agree

File: docs/content/querying/lookups.md [hunk 1]
```diff
@@ -28,6 +28,263 @@ and such data belongs in the raw denormalized data for use in Druid.
 Very small lookups (count of keys on the order of a few dozen to a few hundred) can be passed at query time as a "map" 
 lookup as per [dimension specs](../querying/dimensionspecs.html).
 
-For lookups defined cluster-wide rather than embedded in the query, please look at the [namespaced lookup extension](../development/extensions-core/namespaced-lookup.html).
-
 For other additional lookups, please see our [extensions list](../development/extensions.html).
+
+Dynamic configuration (EXPERIMENTAL)
```
Comment (fjy) [inline docs/content/querying/lookups.md]: can we use the note-caution div to indicate experimental?

**Comment by fjy** [comment]
@drcrallen this will have to be backported

**Comment by drcrallen** [comment]
Dynamic config is not an extension

File: docs/content/querying/lookups.md (hunk 1 above)
Comment (drcrallen) [inline docs/content/querying/lookups.md]: I can, but don't know the syntax you're looking for, is there another doc that uses that indication?

Comment (drcrallen) [inline docs/content/querying/lookups.md]: ah, found it

**Comment by drcrallen** [comment]
@fjy well, this feature is in 0.9.1 not 0.9.0

**Comment by fjy** [comment]
@drcrallen okay

can we remove it from the 0.9.0 docs then?

**Comment by drcrallen** [comment]
they are not in 0.9.0: https://github.com/druid-io/druid/blob/0.9.0/docs/content/querying/lookups.md

File: docs/content/querying/lookups.md (hunk 1 above)
Comment (drcrallen) [inline docs/content/querying/lookups.md]: fixed

**Comment by gianm** [comment]
@drcrallen is the namespaced lookup extension no longer suggested for use in 0.9.1, in favor of the dynamic config stuff?

**Comment by drcrallen** [comment]
@gianm that is still being developed. @sirpkt beat me to the punch on this part: https://github.com/druid-io/druid/pull/2716 

Once #2716 is fleshed out a little more we'll know for sure. But right now there is not enough change in master to make the change in recommendation.

**Comment by gianm** [comment]
ok, I'm asking because this PR removes references to the namespaced-lookups extension from lookups.md. I feel like we should have some stuff on both lookups.md and the namespaced lookup extension docs that clarify the situation for readers.

**Comment by gianm** [comment]
@drcrallen In other words: I am happy with this PR to the degree that you are happy with how lookups.md and namespaced-lookup.md make it clear to readers what the options are for cluster-wide server-side query time lookups, and what the most recommended option is.

**Comment by drcrallen** [comment]
@gianm ah good point, I read

> For lookups defined cluster-wide rather than embedded in the query, please look at the [namespaced lookup extension](../development/extensions-core/namespaced-lookup.html).

and thought of the cluster wide config. I didn't think of the static config in the extension. I'll add that line back in.

**Comment by drcrallen** [comment]
@gianm changed to

>For static lookups defined in `runtime.properties` rather than embedded in the query, please look at the experimental [namespaced lookup extension](../development/extensions-core/namespaced-lookup.html).

**Comment by gianm** [comment]
:+1: looks good to me, thanks for catching this.

**Pull request merged**